pkg/distributor/distributor.go

```diff
@@ -236,7 +237,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	cfg.DistributorRing.RegisterFlags(f, logger)
 	cfg.RetryConfig.RegisterFlags(f)
 
-	f.IntVar(&cfg.MaxRecvMsgSize, "distributor.max-recv-msg-size", 100<<20, "Max message size in bytes that the distributors will accept for incoming push requests to the remote write API. If exceeded, the request will be rejected.")
+	f.IntVar(&cfg.MaxRecvMsgSize, "distributor.max-recv-msg-size", 80<<20, "Max message size in bytes that the distributors will accept for incoming push requests to the remote write API. If exceeded, the request will be rejected.")
```

Comment:
Why was this default value changed? That seems unrelated to this PR.

Reply:
Yep, I was going to ask the same as pstibrany.

Reply:
correct in 1ba1bcb4854d016dfada97077fc1696ea23c84e3, the 80M is for otlp instead.